metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -16,11 +16,11 @@
       type: keyword
       description: >
         Name of Kibana's internal index
-    - name: host.name
+    - name: service.address
```

Comment:
I don't think this will do what you expect as it will place it under the `kibana.stats` prefix. Instead the name should stick around with `type: alias` and `migration: true`. Same for the others in this list.

Best regenerate the `data.json` files then we see if the outcome is as expected.

Reply:
I regenerated the `data.json` file for the `kibana/stats` metricset now.

BTW, I tried updating the `data.go` code to set `service.address` instead of `host.name`. However, it seems that something runs after the `data.go` code and overrides the `service.address` in the resulting event.

Reply:
@ycombinator Good idea about service.address. Perhaps we can tackle this in a separate issue / PR?

Reply:
A separate PR that does not override fields if a module has already set them sounds good. So for this PR I'm *NOT* going to try and set `service.address` in the metricset's `data.go` code, otherwise it will be misleading. Agreed (just making sure we're on the same page here)?